packages/tensorflowc/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Tensorflowc(Package):
```

Comment:
Should we contribute this to spack? It seems very useful to many spack users.

Reply:
Hm I am not sure if this is worth contributing. Not my 1st priority to be honest

Reply:
I understand. Nice package though 👍